frontend/src/views/Dashboard.vue

```diff
@@ -14,12 +14,10 @@
 import { Component, Vue, Watch } from "vue-property-decorator";
 import MainLayout from "@/views/MainLayout.vue";
 import _ from "lodash";
-
```

Comment:
After your edits this file has only useless changes like removing the empty lines. I don't like irrelevant changes in topic branches. So please remove this. But don't do it by another commit, just rewrite the history of this branch. You can squash all the commits into one, in this case splitting changes across several commits is not valuable as you're working on one small feature.